sys/kern/imgact_elf.c

```diff
@@ -597,11 +597,13 @@ __elfN(build_imgact_capability)(struct image_params *imgp,
 		end = MAX(end, seg_addr + seg_size);
 	}
 
+#ifdef __ELF_CHERI
 	if (hdr->e_type == ET_EXEC && !is_aligned(start + rbase,
-	    CHERI_REPRESENTABLE_ALIGNMENT(end - start))) {
+		CHERI_REPRESENTABLE_ALIGNMENT(end - start))) {
 		/*
-		 * We can't change the load address for position dependent
-		 *  executables, so we have to give up and report an error.
+		 * We can't change the load address for position
+		 * dependent executables, so we have to give up and
+		 * report an error.
```

Comment:
If you want to reformat the existing code (even if just comments), please do that separately from adding the ifdef?